### ISSUE TITLE
About Data segmentation

### ISSUE THREAD

**Issue by AI4S-Ritsuka** [MIILab-MTU/ICA_NJ_BinarySeg#1]
Hello, Thanks for your Dataset, In your excellent work you mentioned:
 "_**At the training stage, we split the 99 patients into a training set, an
internal validation set, and an external test set, containing 60%, 10%
and 30% of the patients respectively. As a result, 196, 28 and 90 images
were grouped into the training set, validation set, and external test set,
respectively.**_ "
This mean your random spilt the dataset according to the patients' number? the patient number is nj_XXX_L... ?
I noticed that the png files more than 314, if I want to use it to train my model how can I spilt these data?
Thanks for your excellent work !!!
